renderer/components/Onboarding/Steps/components/SkipBackupsDialog.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Flex, Box } from 'rebass'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import { withFieldState } from 'informed'
+import Warning from 'components/Icon/Warning'
+import { Dialog, Text, Heading, Button, Checkbox, Form, DialogOverlay } from 'components/UI'
+import { useCloseOnUnmount } from 'hooks'
+import messages from './messages'
+
+const DialogWrapper = ({ intl, isOpen, onSkip, onCancel }) => {
+  useCloseOnUnmount(isOpen, onCancel)
+
+  if (!isOpen) {
+    return null
+  }
+  const checkboxFieldName = 'actionACK'
+
+  // bind button disabled state to a form field or ignore it if we are dealing with a remote
```

Comment:
looks like comment carried over from delete wallet dialog: `dealing with a remote wallet...`. Should be updated.

Reply:
lol, i even fixed it yesterday but forget to push